monai/transforms/compose.py

```diff
@@ -123,6 +177,9 @@ def __init__(
         map_items: bool = True,
         unpack_items: bool = False,
         log_stats: bool = False,
+        lazy_evaluation: bool | None = None,
+        overrides: dict | None = None,
+        override_keys: Sequence[str] | None = None,
```

Comment:
need to provide some good default when `lazy_evaluation=True`

Reply:
The default behaviour should be that no overrides are performed, no? The overrides are there for where you have incompatible sets of modes / padding_modes and so forth. I'm working on the PR to reduce the number of cases where this is an issue.